mycore-base/src/main/java/org/mycore/datamodel/classifications2/impl/MCRCategLinkServiceImpl.java

```diff
@@ -63,9 +62,7 @@
  */
 public class MCRCategLinkServiceImpl implements MCRCategLinkService {
 
-    private static MCRHIBConnection HIB_CONNECTION_INSTANCE;
```

Comment:
Wouldn't it be better to this MCRHIBConnection removal in another (separate, smaller) PULL-request?

Reply:
We could make a separate "remove old hibernate code" ticket